### PR TITLE
Add info about revisions

### PR DESCRIPTION
**Pull request by luisherranz**
This PR instructs users that revisions need to be activated in order to retrieve the latest preview content from the REST API.

### CODE REVIEW

**Comment by mburridge** [comment]
@luisherranz Thanks for adding this. Do you think it might be an idea to link to the docs for `register_post_type` as well, so that developers can add support for revisions to CPTs.
https://developer.wordpress.org/reference/functions/register_post_type/#supports

File: docs-api/frontity-plugins/embedded-mode.md [hunk 1]
```diff
@@ -64,6 +64,10 @@ Installation is a three step process: Download → Install → Activate.
 
 3. **Activate** the plugin by locating it in the Plugins list and clicking on 'Activate'.
 
+{% hint style="warning" %}
```
Comment (mburridge) [inline docs-api/frontity-plugins/embedded-mode.md]: Would this hint be better placed just below lines 48/49 above?

Comment (luisherranz) [reply docs-api/frontity-plugins/embedded-mode.md]: Done.

**Comment by luisherranz** [comment]
> Do you think it might be an idea to link to the docs for register_post_type as well, so that developers can add support for revisions to CPTs.

Sure. Done 🙂

**Review by mburridge — APPROVED** [review]

**Comment by mburridge** [comment]
Approved and merged, thanks @luisherranz

**Pull request merged**